src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileServiceSpec.scala

```diff
@@ -737,7 +737,7 @@ class FileServiceSpec extends AnyFlatSpec with MockitoSugar with Matchers with S
     val expectedSize = 56
     metadataRows.size should equal(expectedSize)
     staticMetadataProperties.foreach(prop => {
-      metadataRows.count(r => r.propertyname == prop.name && r.value == prop.value) should equal(5)
+      metadataRows.count(r => r.propertyname == prop) should equal(5)
```

Comment:
These can all be simplified to `metadataRows.count(_.propertyname == prop) should equal(5)`